local-tests/send-runtime/rust-toolchain

```diff
@@ -1 +0,0 @@
-nightly-2021-10-24
```

Comment:
Are we removing it as nightly is not needed for send-runtime and fork-off?

Reply:
All rustup tools will walk up the directory tree in search for the rust-toolchain file, as stated in the very first section of https://rust-lang.github.io/rustup/overrides.html
My point is to leave only a single configuration file that will specify the toolchain version.

Reply:
Oh, cool stuff